### PR TITLE
Adding additional log for support multiple projects from tracker.

### PR DESCRIPTION
**Pull request by hd-lj**
Please test this out...this is the just the multiple project support...I've split all the changes i've made into new branches.

### CODE REVIEW

**Comment by hd-lj** [comment]
I pulled out just the multiple tracker projects part of my changes and some styling fixes. I made another fix on that same branch that. I didn't feel should be included since it was for the slack module. I will do second pull request for that since it definitely needs additional testing in that module. Please manually test. We did test manually. But it's always good to test on another persons machine. Or even a eat your own dog food approach with some automation with travis and pivotal tracker.

**Comment by mkdillard** [comment]
I also think overall it looks okay.  I do have a concern here https://github.com/homedepot/flow/pull/51/files#diff-85126f0265dac5f79c7a24f99fe104a9R126

I feel like this implies an assumption that story ids are unique across all projects, since we break out of the loop at the first 200 response received.  Do we know that story ids are unique across all projects?

**Comment by WRMilling** [comment]
> ... Do we know that story ids are unique across all projects?

They should be, yes. For instance, if you link someone to a story through their UI, it doesn't include the project id, simply creates a link like `https://www.pivotaltracker.com/story/show/123456789`.

File: flow/projecttracking/tracker/tracker.py [hunk 1]
```diff
@@ -76,34 +96,38 @@ def _retrieve_story_detail(self, story_id):
...
-        headers = {'Content-type': 'application/json', 'Accept': 'application/json', 'X-TrackerToken': Tracker.token}
+        tracker_story_details = []
+        json_data = None
+        resp = None
 
-        commons.print_msg(Tracker.clazz, method, tracker_story_details_url)
+        if Tracker.project_ids is not None:
+            for project_id in Tracker.project_ids:
+                tracker_story_details.append(
+                    {'url': Tracker.tracker_url + '/services/v5/projects/' + project_id + '/stories/' + story_id})
```
Comment (WRMilling) [inline flow/projecttracking/tracker/tracker.py:106]: Not sure if there are any gotchas, but we can probably use the following API to get around having to loop through all the `project_id`s

```
export TOKEN='your Pivotal Tracker API token'

curl -X GET -H "X-TrackerToken: $TOKEN" "https://www.pivotaltracker.com/services/v5/stories/559"
```

From: https://www.pivotaltracker.com/help/api/rest/v5#Stories (Does not let me link directly to the specific endpoint, scroll down through this section).

**Review by WRMilling — APPROVED** [review]

**Pull request merged**